android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -299,11 +312,10 @@ public static void setUsingTestConfiguration(boolean shouldUseTestConfiguration)
         testConfigurationFlag = shouldUseTestConfiguration;
     }
 
-    public void clearTestUpdates() {
-        if (isUsingTestConfiguration()) {
-            codePushPackage.clearTestUpdates();
-            removePendingUpdate();
-        }
+    public void clearUpdates() {
+        codePushPackage.clearUpdates();
```

Comment:
In Android, yes getBundleUrl will get called, but it would resolve to the package file if there is an update installed, so clearUpdates() wouldn't get called. The only time it would get called is if a new binary is applied, which I think makes sense even in the dev scenario. What do you think?